uaclient/system.py

```diff
@@ -154,28 +156,33 @@ def get_platform_info() -> Dict[str, str]:
         "type": "Linux",
     }
 
-    version = os_release["VERSION"]
-    # Strip off an LTS point release (20.04.1 LTS -> 20.04 LTS)
-    version = re.sub(r"\.\d LTS", " LTS", version)
+    version = re.sub(r"\.\d LTS", " LTS", os_release["VERSION"])
     platform_info["version"] = version
+    series = os_release.get("VERSION_CODENAME")
+    release = os_release.get("VERSION_ID", "")
```

Comment:
Let's be consistent with our fallback values - I'm okay with either None or ""
```diff
-series = os_release.get("VERSION_CODENAME")
+series = os_release.get("VERSION_CODENAME", "")
```

Same down below where we use `.get("series")`